README.md

```diff
@@ -14,10 +14,13 @@ More details at [kilvish.in](https://kilvish.in)
 1. Setup Flutter on your PC
    - [Download flutter](https://docs.flutter.dev/get-started/install)
    - [Setup Editor](https://docs.flutter.dev/get-started/editor?tab=vscode)
-   - Run `flutter doctor` in your PC terminal to figure out anything that flutters need that is not installed on your PC
+   - Run `flutter doctor` in your PC terminal to figure out anything that flutter needs that is not installed on your PC.
 2. Run code
-   - `flutter pub get` to install dependencies. Your editor will do it by default once you have cloned the repository & opened it in the editor
-   - `flutter run` will launch the app in debug mode. You can also launch it from Run command of your editor.
+   - Run `git clone https://github.com/tamraj-kilvish/kilvish.git` in your PC terminal to clone the repository.
+   - Navigate to the directory of the cloned repository in your PC terminal.
+   - `flutter pub get` to install dependencies.
```

Comment:
I would recommend to remove flutter pub get line. Reason is it is doing by default so no need to be mention in detail.

Reply:
Keep flutter run simple.

Reply:
Not everyone uses Visual Studio. I guess it doesn't hurt to keep it.

Reply:
Every ide does it by default. Dart has some extentions that is installed by default that runs flutter pub command.

Reply:
Everyone doesn't use an IDE. For instance, I'm using a lightweight editor right now. Again, I don't see any problem with keeping the line - it doesn't take much effort to run the command.